content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Resource Types
+content_type: reference
+---
+
+<!-- overview -->
+
+This page provides information on the different resource types you can request, built into Kubernetes.
+
+<!-- body -->
```

Comment:
This is a short overview IMO. Best to merge it with the paragraphs that follow. That way “CPU” is the first part of the page body.